src/keys/CompositeKey.cpp

```diff
@@ -97,19 +97,27 @@ QByteArray CompositeKey::rawKey() const
  * as a challenge to acquire their key contribution.
  *
  * @param transformSeed transform seed to challenge or nullptr to exclude challenge-response components
+ * @param ok true if challenges were successful and all key components could be added to the composite key
  * @return key hash
  */
-QByteArray CompositeKey::rawKey(const QByteArray* transformSeed) const
```

Comment:
Honestly I would rather we return bool and have a QByteArray& result input. This style makes coding very janky. It also is completely different then how the transformKey function is written.

Reply:
We have both styles.

Reply:
Actually, I would prefer to leave it as-is. The abstract Key class has a pure-virtual rawKey() function which returns the raw key as byte array. This function is just an overload and should behave similarly. Changing both requires a lot of (unnecessary) code changes and changing only this one makes it inconsistent. Also I think needing lvalues for the actual key is more annoying than (optionally) needing an lvalue for the `ok` value (@TheZ3ro `ok` is the name we use everywhere else for this kind of stuff).

Reply:
Defer the api change to 2.4